pages/use-solid/apps.md

```diff
@@ -6,6 +6,12 @@ permalink: /use-solid/apps
 
 These Solid-compatible apps will store your data on [your own Pod]({{site.baseUrl}}/use-solid).
 
+The criteria for an app to be Solid-compatible are: 
+* If identifying users is necessary, they must be able to login using their WebID and pointing to the Identity Provider of their choice
+* Data consumed by the app should be fetched from Solid Pods
```

Comment:
I would remove this requirement, or make it optional; data can come from many places.